File: app/src/main/java/com/example/myapplication/ui/choix/ChoixViewModel.java
package com.example.myapplication.ui.choix;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ChoixViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public ChoixViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is tables fragment");

    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/example/Voicings/ui/choix/ChoixViewModel.kt
package com.example.Voicings.ui.choix

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ChoixViewModel : ViewModel() {
    private val mText = MutableLiveData<String>()

    init {
        mText.value = "This is tables fragment"
    }

    val text: LiveData<String>
        get() = mText
}